packages/manhattan/src/index.js

```diff
@@ -196,21 +168,21 @@ const ManhattanPlot = ({
 ManhattanPlot.propTypes = {
   data: PropTypes.arrayOf(PropTypes.shape({
     snp: PropTypes.string.isRequired,
-    chromosome: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 'X', 'Y']).isRequired,
+    chromosome: PropTypes.oneOf(HUMAN_CHROMOSOMES).isRequired,
```

Comment:
We've had problems before (particularly with Elasticsearch mappings) from mixing numeric and string types for chromosomes.